aldjemy/orm.py

```diff
@@ -125,7 +125,7 @@ def _extract_model_attrs(metadata, model, sa_models):
 
 def prepare_models():
     metadata = get_meta()
-    models = [model for model in get_django_models() if not model._meta.proxy]
+    models = [model for model in get_all_django_models() if not model._meta.proxy]
```

Comment:
Why do we need this change? Can the difference been tested?

Reply:
I need `construct_models` to return all of the models that were generated, including autogenerated through tables, because I'm doing some manual work with those tables. I should be able to get it tested.

Reply:
I added a bit to the tests to check on the through model to make sure it has an `sa` attribute like all the other models.